docs/examples/amazon-dynamodb-sessions.md

```diff
@@ -0,0 +1,92 @@
+Griptape provides [Conversation Memory](../griptape-framework/structures/conversation-memory.md) as a means of persisting conversation context across multiple Structure runs.
+By default, Griptape's Conversation Memory does not provide a Coversation Memory Driver which means the conversation history is lost after the Structure is finished running.
```

Comment:
"If you provide it with a suitable a Driver, the memory of the previous conversation can be preserved between invocations (runs?) of a Structure, giving it additional context for how to respond."